RealLifeGTAV/dist/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "N0chteil ^^",
+    "id": "495901098926669825"
+  },
+  "contributors": [
+    {
+      "name": "N0chteil",
+      "id": "495901098926669825"
+    }
+  ],
+  "url": [
+    "reallifegtav.de",
+    "www.reallifegtav.de"
+  ],
+  "description": {
+    "en": "Real Life GTAV is a german FiveM roleplay server with custom scripts and more!",
+	  "de": "Real Life GTAV ist ein deutscher FiveM Roleplay Server mit eigenen Scripts und mehr!"
+  },
+  "service": "Real Life GTAV",
+  "version": "1.0.0",
```

Comment:
service name and folder name must be 100% the same

Reply:
ok